nemo/collections/asr/data/audio_to_text_dali.py

```diff
@@ -278,7 +268,7 @@ def __init__(
             self.pad_value = params['pad_value'] if 'pad_value' in params else 0.0
 
         with self.pipe:
-            audio, transcript = dali.fn.nemo_asr_reader(
+            audio, indices = nemo_asr(
```

Comment:
might be a matter of personal preference, but I think that it will read better if it was `dali.fn.readers.nemo_asr` or `fn.readers.nemo_asr` so that it's visible that this is a DALI operator.